2021.01.12

zinna-choi opened the pull request.

zinna-choi committed:
file upldad init

ziponia committed:
progress upload

ziponia committed:
progress fileupload

ziponia committed:
progress fileupload

ziponia committed:
update env for vercel

zinna-choi committed:
Add Change User Profile Image

zinna-choi committed:
Merge branch 'dev' into fileupload

ziponia committed:
fix: compile error

ziponia commented on package.json:
```diff
@@ -14,16 +14,19 @@
     "next": "latest",
     "next-auth": "^3.1.0",
     "next-connect": "^0.9.1",
+    "node-libs-browser": "^2.2.1",
```
https://www.npmjs.com/package/node-libs-browser

라이브러리 지원종료됨.. 재확인

ziponia commented on src/components/FileUpload/index.tsx:
```diff
@@ -0,0 +1,46 @@
+    }
+  },
+}
+const FileUpload: React.FC<FileUploadProp> = (props) => {
```
`FileUploadProp` -> `FileUploadProps`

ziponia commented on src/components/FileUpload/index.tsx:
```diff
@@ -0,0 +1,46 @@
+const { Dragger } = Upload
+// const [fileList, updateFileList] = useState([])
+
+export type FileUploadProp = DraggerProps & {}
```
`FileUploadProp` -> `FileUploadProps`

ziponia commented on src/components/FileUpload/index.tsx:
```diff
@@ -0,0 +1,46 @@
+const FileUpload: React.FC<FileUploadProp> = (props) => {
+  return (
+    <div>
+      <Dragger {...props}>
```
`const { children, ...rest } = props`
형태로, react children props 는 전달되지 못하도록

ziponia commented on src/pages/user/mypage.tsx:
```diff
@@ -21,23 +21,34 @@ import {
+import { AnyARecord } from "dns"
+import MyPageTemplate from "../../templates/MyPageTemplate"
+import DefaultModal from "../../components/Modal"
+import FileUpload from "../../components/FileUpload"
```
path 는 `@covid/` 로 설정 되었습니다.

예)
```
import MyPageTemplate from "../../templates/MyPageTemplate" (x)
import MyPageTemplate from "@covid/templates/MyPageTemplate" (o)
```

ziponia commented on src/pages/user/mypage.tsx:
```diff
@@ -130,7 +141,9 @@ type Props = {
 const MyPage: AppPageProps<Props> = (props) => {
   const screens = useBreakpoint()
   const isMobileScreen = screens.xs && !screens.md
```
변수는, useState 보다 아래로 선언

예)
```
const [counter, setCounter] = useCount(0)
const isMobileScreen = screens.xs && !screens.md
```

ziponia committed:
[fix] ant upload components warning

https://github.com/ant-design/ant-design/blob/master/components/upload/Upload.tsx

zinna-choi committed:
Improve file upload function

zinna-choi committed:
Merge branch 'fileupload' of https://github.com/ziponia/covid-com into fileupload
